### ISSUE TITLE
Configuration to adjust spawn rates

### ISSUE THREAD

**Issue by Calendis** [ExcessiveAmountsOfZombies/Croptopia#294]
### Make sure you are not opening a duplicate.

- [X] [I have searched the issue tracker](https://github.com/ExcessiveAmountsOfZombies/Croptopia/issues?q=) and did not find an issue describing my bug.

### Description of your suggestion.

Version: `1.7.4 Fabric`

I would like to adjust the spawn rates of fruit trees. I see I can adjust the biomes, which is great, but I just want to tweak the rates

**Comment by flumignano** [comment]
Agreed. Fruit trees are very common when they should be very rare IMO and I wish I could adjust the rates because they currently make survival and villager trading too easy (for apples)!

**Comment by ChristopherTheBoss** [comment]
Yeah I pretty much made a duplicate issue my bad
